recipes/lbapcommon/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "LbAPCommon" %}
+{% set version = "0.0.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/LbAPCommon-{{ version }}.tar.gz
+  sha256: 0713326ec22bdba7292c51dac23a3ebb4747a3bdddbe8f5f51898209df54d045
+
+build:
+  number: 0
+  noarch: python
```

Comment:
We'll need to make sure this package builds on noarch as 2.7, right? Is there a good way to enforce that?

Reply:
cc @conda-forge/core since I am not 100% sure here

Reply:
It could be forced with:

```yaml
requires:
  host:
    - python 2.7
  run:
    - python !=3.0,!=3.1,!=3.2,!=3.3,!=3.4,!=3.5
```

but I don't think it's needed as it's being tested with multiple Python versions upstream.

Reply:
mmk